src/Analyser/ResultCache/ResultCacheManager.php

```diff
@@ -159,7 +159,7 @@ public function restore(array $allAnalysedFiles, bool $debug, bool $onlyFiles, ?
 				continue;
 			}
 			if (!is_file($extensionFile)) {
-				if ($output->isDebug()) {
+				if ($output->isVerbose()) {
```

Comment:
Should be `isVeryVerbose()` in places where you still left `isVerbose()`.

Reply:
Missed this one, thx

Reply:
Fixed. Double checked, last one in verbose mode is the "Result cache restored", would you keep it?